server/methods/importProducts.js

```diff
@@ -0,0 +1,140 @@
+  prod.metafields = [];
+
+  _.each(features, function (feature) {
+    let metafield = {};
+    metafield.key = 'feature';
+    metafield.value = feature;
+    prod.metafields.push(metafield);
+  });
+
+  prod.metafields.push({key: 'aboutVendor', value: aboutVendor});
+
+  _.each(colors, function (color) {
+    let variant = {};
+    variant._id = Random.id();
+    variant.type = 'variant';
+    variant.title = color.trim();
+    variant.optionTitle = 'color';
+    variant.price = '1.00';
+    variant.pricePerDay = '1.00';
+    variant.inventoryQuantity = sizes.length;
+    variant.taxable = true;
+    variant.weight = 1;
+    prod.variants.push(variant);
+
+    _.each(sizes, function (size) {
+      let childVariant = {};
+      childVariant._id = Random.id();
+      childVariant.parentId = variant._id;
+      childVariant.sku = generateSku(product, color, size);
+      childVariant.color = color.trim();
+      childVariant.size = size.trim();
+      childVariant.title = 'size';
+      childVariant.optionTitle = size.trim();
+      childVariant.inventoryQuantity = 1;
+      childVariant.price = '1.00';
+      childVariant.pricePerDay = '1.00';
+      childVariant.taxable = true;
+      childVariant.weight = 1;
+
+      prod.variants.push(childVariant);
+    });
+  });
+
+  return prod;
+}
+
+function setupBundleDocument(bundle) {
+  // let jacketSizes = product.body_html.split(':jacketsizes:')[1].split(','); // Array of jacket sizes - empty if no jacket.
+  // let pantSizes = product.body_html.split(':pantsizes:')[1].split(','); // Array of pants sizes - empty if no pants
+  // let gloveSizes = product.body_html.split(':glovesizes:')[1].split(','); // Array of glove sizes - empty if no gloves
+  // let goggleTypes = product.body_html.split(':goggletypes:')[1].split(','); // Array of goggle types - empty if no goggles or single type.
+  // let goggleStyle = product.body_html.split(':goggles:')[1]; // Goggle style - for packages - premium mirrored vs standard
+  // let gloves = product.body_html.split(':gloves:')[1]; // Glove style - for packages - premium goretex vs standard
+}
+
+Meteor.methods({
+  importProducts: function (productType = 'Jacket', createdAtMin = '2015-09-01') {
+    check(createdAtMin, String);
+    check(productType, String);
+    const shopifyCredentials = ReactionCore.Collections.Packages.findOne({name: 'reaction-shopify-products'}).settings.shopify;
+    const Products = ReactionCore.Collections.Products;
+    const apikey = shopifyCredentials.key;
+    const password = shopifyCredentials.password;
+    const domain = 'https://' + shopifyCredentials.shopname + '.myshopify.com';
+    const query = '/admin/products.json';
+    const fields = 'id,title,body_html,vendor,product_type,handle,tags';
+
+    let res = HTTP.call('GET', domain + query, {
+      params: {
+        fields: fields,
+        product_type: productType,
+        created_at_min: createdAtMin
+      },
+      auth: apikey + ':' + password
+    });
+
```

Comment:
For my knowledge is there a benefit of declaring `const domain = ` rather than just placing that into the HTTP call?